### DISCUSSION TITLE
Translate from the meta function?

### DISCUSSION THREAD

**Discussion by davidesigner** [sergiodxa/remix-i18next#188] [Q&A] [answered]
Hi there,

What do you recommend to use translation key in the meta?

For example, we could want to index an english and a french version of a page on Google with right meta.

i.e. `/en/blog-post` with a meta title: "My english title" & `/fr/article-blog` with a meta title: "Mon article de blog Français"

Based on the URL, we can know if we want to load the english or the french version, or the fallbackLanguage if nothing appears on the URL.

Some solutions I thought:
- Send the meta object translated from the loader data to the meta function (looks like a hack).
- Send the `t` function from the loader to the meta (looks like a smaller hack).

Do you have any better idea?

**Comment by sergiodxa** [comment, marked as answer]
You should send the translated messages, that's why the package gives you getFixedT to use inside loader.

The `t` function can't be sent from the loader to meta because MetaFunction run's client-side too so the data it receives will need to be serialized and functions can't be serialized.

**Reply by davidesigner** [reply]
Thanks for your feedback Sergio!
